client/cs/src/client.csproj

```diff
@@ -19,7 +19,7 @@
     <NoWarn>CS8629</NoWarn>
 
     <!-- This is our own property that determines the native library name. -->
-    <TpNativePrefix>$(MSBuildThisFileDirectory)..\..\..\target\debug\libunity_states</TpNativePrefix>
+    <TpNativePrefix>$(MSBuildThisFileDirectory)..\..\..\target\libunity_states</TpNativePrefix>

```

Comment:
why was `debug` removed? My generated targets all appear under `debug`

Reply:
This allows us to choose between debug and release builds without changing the target path of the symlinks in the UnityStates demo.